core/controllers/acl_decorators_test.py

```diff
@@ -2115,6 +2115,42 @@ def test_guest_cannot_publish_skill(self):
             'You must be logged in to access this resource.')
 
 
+class AccessUserSkillMasteryTests(test_utils.GenericTestBase):
+    """Tests for decorator can_access_user_skill_mastery."""
+    viewer_username = 'viewer'
+    viewer_email = 'viewer@example.com'
+
+    class MockHandler(base.BaseHandler):
+        GET_HANDLER_ERROR_RETURN_TYPE = feconf.HANDLER_TYPE_JSON
+
+        @acl_decorators.can_access_user_skill_mastery
+        def get(self):
+            self.render_json({})
+
+    def setUp(self):
+        super(AccessUserSkillMasteryTests, self).setUp()
+        self.signup(self.viewer_email, self.viewer_username)
+
+        self.mock_testapp = webtest.TestApp(webapp2.WSGIApplication(
+            [webapp2.Route('/mock_user_skill_mastery', self.MockHandler)],
```

Comment:
Maybe store '/mock_user_skill_mastery' as a constant (like viewer_email above) and use it throughout the test

Reply:
Other mock URLs are not stored as constants in this file, so maybe it's better to keep them consistent.